### PR TITLE
Podcast episode 46 — Nextflow meets WDL

### PR DESCRIPTION
**Pull request by vdauwera**
Need to add Youtube ID once video file is final and uploaded

### CODE REVIEW

File: src/content/podcast/2024/ep46_nextflow_vs_wdl.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+youtubeid: qeJqdV_d8vI
+image: /img/podcast_ep46.png
+tags: Nextflow
+author: Developer advocates
+icon: logo_podcast_channels.jpg
+---
+
+In this episode of Channels, the Nextflow podcast, host Geraldine Van der Auwera from Seqera and her colleague Ben Sherman dive into the differences and similarities between Nextflow and the [Workflow Description Language](https://openwdl.org/) (WDL). They discuss the origins and development of WDL at the Broad Institute, the challenges of working with different workflow languages, and how Nextflow's channel-based data flow model compares to WDL's approach. The conversation covers key features, type systems, and runtime specifications, highlighting the strengths and limitations of both languages. Ben hints at upcoming improvements and enhancements for Nextflow, aimed at making the language more robust and user-friendly. The episode offers valuable insights for bioinformaticians and developers navigating the complexities of workflow management.
+
+<!-- end-archive-description -->
```
Comment (ewels) [inline src/content/podcast/2024/ep46_nextflow_vs_wdl.md:17]: Side note: Generally don't need these, it'll cap the description after the first paragraph automatically (or a certain character limit). This comment is only needed if you want it to be capped early.

But doesn't hurt to include either..

**Review by ewels — APPROVED** [review]
Great!

File: src/content/podcast/2024/ep46_nextflow_vs_wdl.md (hunk 1 above)
Comment (vdauwera) [reply src/content/podcast/2024/ep46_nextflow_vs_wdl.md:17]: Oh I was wondering why they weren't used consistently in the past ones

File: src/content/podcast/2024/ep46_nextflow_vs_wdl.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+### Introduction to WDL
+
+WDL, or Workflow Description Language, was developed at the Broad Institute starting around 2014. This language emerged during a period when existing workflow solutions were struggling to scale with increasing data volumes and the need for greater reproducibility. The Broad Institute, a nonprofit research institution in Cambridge, MA, needed a more effective software stack for its genome analysis pipeline, which was moving to the cloud.
+Several workflow management tools were considered, including Queue (integrated with the GATK), Firehose, and Zamboni. The CWL (Common Workflow Language) which was emerging at the time was also reviewed but deemed too machine-oriented. This led to the development of WDL as a user-friendly, cloud-native DSL (domain-specific language) designed for high reproducibility. Today, WDL is an open-source project supported by two main engines: [Cromwell](https://github.com/broadinstitute/cromwell) (focused on high-scale execution) and [MiniWDL](https://github.com/mlin/miniwdl) (geared towards prototyping).
```
Comment (ewels) [inline src/content/podcast/2024/ep46_nextflow_vs_wdl.md:44]:
```suggestion
This led to the development of WDL as a user-friendly, cloud-native DSL (domain-specific language) designed for high reproducibility. Today, WDL is an open-source project supported by two main engines: Cromwell (focused on high-scale execution) and MiniWDL (geared towards prototyping).
```

**Pull request merged**